server/src/main/java/org/jetbrains/bsp/bazel/server/bsp/config/ServerArgsProjectViewProvider.java

```diff
@@ -1,28 +1,47 @@
 package org.jetbrains.bsp.bazel.server.bsp.config;
 
 import com.google.common.collect.ImmutableList;
-import java.nio.file.Paths;
+import io.vavr.control.Try;
+import java.nio.file.Path;
 import java.util.Arrays;
+import java.util.Optional;
 import org.jetbrains.bsp.bazel.projectview.model.ProjectView;
 import org.jetbrains.bsp.bazel.projectview.model.ProjectViewProvider;
-import org.jetbrains.bsp.bazel.projectview.model.sections.specific.DirectoriesSection;
-import org.jetbrains.bsp.bazel.projectview.model.sections.specific.TargetsSection;
+import org.jetbrains.bsp.bazel.projectview.model.sections.ProjectViewBazelPathSection;
+import org.jetbrains.bsp.bazel.projectview.model.sections.ProjectViewTargetsSection;
+import org.jetbrains.bsp.bazel.projectview.parser.ProjectViewDefaultParserProvider;
 
+@Deprecated
```

Comment:
Is there an alternative? Currently all implementations of `ProjectViewProvider` are deprecated.